doc/manual/cookbooks/prescribed_velocity/prescribed_velocity.cc

```diff
@@ -0,0 +1,223 @@
+#include <deal.II/base/parameter_handler.h>
```

Comment:
I ask because it is so much easier to get the manual to be correct and in sync with the file in the cookbooks/ directory if there are not two copies that one always has to update at the same time if anything should ever change.